pallets/manta-pay/src/rpc.rs

```diff
@@ -32,7 +32,12 @@ pub trait PullApi {
     /// Returns the update required to be synchronized with the ledger starting from
     /// `checkpoint`.
```

Comment:
actually this doccomment is probably more important to annotate with the new params